backend/expense/src/main/java/org/egov/digit/expense/util/MdmsUtil.java

```diff
@@ -48,6 +49,7 @@ public Map<String, Map<String, JSONArray>> fetchMdmsData(RequestInfo requestInfo
             mdmsResponse = mapper.convertValue(response, MdmsResponse.class);
         }catch(Exception e) {
             log.error("Exception occurred while fetching category lists from mdms: ",e);
+			throw new CustomException("MDMS_ERROR", "Exception occurred while fetching mdms data");
```

Comment:
What master, what error ? inject those values in error message

Reply:
done